nengo_gui/tests/test_data_to_csv.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver import ActionChains
+import time
+import pytest
+from nengo_gui import conftest
+from nengo_gui import testing_tools as tt
+import os
+
+def test_data_to_csv(driver):
+	tt.reset_page(driver)
+
+	tt.update_editor(driver,'''
+import nengo
```

Comment:
I definitely agree with @Seanny123 for multiline strings in JS, but in Python using the triple quotes is better, I believe. In addition to the \n line endings, the triple quotes here maintains the indentation in the string itself, which is necessary for Python. In JS, the indentation makes it readable in the source, but the generated HTML doesn't have that indentation. Which is fine, because we never look at the generated HTML, but for Python it makes a functional difference.